specs/logging_change.adoc

```diff
@@ -0,0 +1,152 @@
+// vim: tw=avored-markdown/preview.ht79
+
+# Changes in tendrl modules after introduction of new logging framework
+
+Since a new logging framework is being introduced (https://github.com/Tendrl/specifications/pull/94), after its implementation
+there will be requirement of changes in all the modules of Tendrl where logging is used.
+This file proposes all those changes and how they will be made.
+
+
+## Problem description
+
+Many modules in Tendrl have a functionality of logging different types of situations/use cases such as logging (info, errors, warning etc.).
+Till now these were implemented by importing the Python standard library's logging module then calling logger functions to handle the logging.
+After implementation of new logging framework there will be requirement of changes in all the modules of Tendrl where logging is used.
+
+#### Use Cases
+
+Logs and operation logs need be treated as messages.
+
+#### Proposed change
+
+* *Importing logging and calling logger functions* will be replaced by:
```

Comment:
@GowthamShanmugam is already working on this https://github.com/Tendrl/ceph-integration/pull/103

@anmolsachan both of you collaborating on this one?